src/main/java/biblioteka/Calendar.java

```diff
@@ -0,0 +1,354 @@
+
+            int affectedRows = pstmt.executeUpdate();
+            // check the affected rows
+            if (affectedRows > 0) {
+                // get the ID back
+                try (ResultSet rs = pstmt.getGeneratedKeys()) {
+                    if (rs.next()) {
+                        id = rs.getLong(1);
+                    }
+                }catch (SQLException ex) {
+                    System.out.println(ex.getMessage());
+                }
+            }
+        } catch (SQLException ex) {
+            System.out.println(ex.getMessage());
+        }
+        return id;
+    }
+
+    public void insertUsers(List<User> list) {
+        String SQL = "INSERT INTO users(name, surname) "
+                + "VALUES(?,?)";
+        try (
+                PreparedStatement statement = conn.prepareStatement(SQL);) {
+            int count = 0;
+
+            for (User user : list) {
+                statement.setString(1, user.getName());
+                statement.setString(2, user.getSurname());
+
+                statement.addBatch();
+                count++;
+                // execute every 100 rows or less
+                if (count % 100 == 0 || count == list.size()) {
+                    statement.executeBatch();
+                }
+            }
+        } catch (SQLException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public int getUserCount() {
+        String SQL = "SELECT count(*) FROM users";
+        int count = 0;
+
+        try (Statement stmt = conn.createStatement();
+             ResultSet rs = stmt.executeQuery(SQL)) {
+            rs.next();
+            count = rs.getInt(1);
+        } catch (SQLException ex) {
+            System.out.println(ex.getMessage());
+        }
+
+        return count;
+    }
+
+    public void findUserByID(int userID) {
+        String SQL = "SELECT user_id, name, surname "
+                + "FROM users "
+                + "WHERE user_id = ?";
+
+        try (PreparedStatement pstmt = conn.prepareStatement(SQL)) {
+
+            pstmt.setInt(1, userID);
+            ResultSet rs = pstmt.executeQuery();
+            displayUser(rs);
+        } catch (SQLException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public void getUsers() {
+        String SQL = "SELECT user_id, name, surname FROM users";
+
+        try (Statement stmt = conn.createStatement();
+             ResultSet rs = stmt.executeQuery(SQL)) {
+            // display user information
+            displayUser(rs);
+        } catch (SQLException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    private void displayUser(ResultSet rs) throws SQLException {
+        while (rs.next()) {
+            System.out.println(rs.getString("user_id") + "\t"
+                    + rs.getString("name") + " "
+                    + rs.getString("surname"));
+        }
+    }
+
+
+    public int changeUserName(int id, String name) {
```

Comment:
I think it's just better to have basic operations like ADD/MODIFY/DELETE/FIND user. In this case we must add a method for each param

Reply:
Ok, we can assume that for now we will keep only the basic methods for now. Deleted this methods